docs/tutorials/teach-atomist-new-skills.md

```diff
@@ -0,0 +1,103 @@
+Software development does not occur solely on the developer's machine any
+longer. It has become a collaborative task opened to all members of the team
+and interactions are constant.
+
+The Atomist Bot you have enrolled to your team strives to be tailored to what
+makes sense to your team. To do so, you can teach the bot new skills 
+automating common tasks your team routinely performs. Those skills are 
+implemented inside Rug command handlers and this tutorial will show you their 
+powerful, and indeed fun, capabilities.
+
+!!! tip "All you need is Rug"
+    These tutorials assume you have a [Rug project][ugpj] which hosts your
+    new [Rug command handlers][rugcmd]. You can easily add them using the 
+    [AddTypeScriptCommandHandler Rug editor][rugeditor] provided by Atomist.
+
+[ugpj]: /user-guide/rug/projects.md
+[rugcmd]: /user-guide/rug/commands.md
+[rugeditor]: https://github.com/atomist/rug-editors#addtypescriptcommandhandler
+
+## Search StackOverflow using the Bot
+
+StackOverflow is one of the biggest venue for developers to collaborate
+together. It is a rather common place to go to and using Atomist you can let
```

Comment:
Remove `together` - kind of implied. Perhaps `collaborate when creating software`